{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "f021f080-76b3-4513-a08a-d251d5705e57",
   "metadata": {},
   "outputs": [],
   "source": [
    "#https://marcobonzanini.com/2015/01/12/searching-pubmed-with-python/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "330086eb-e796-493f-a8db-e015f6ec1223",
   "metadata": {},
   "outputs": [],
   "source": [
    "#!pip install biopython"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "a7f6ca2d-b8fc-4181-89a7-9fdd2fa8d4e9",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "5a0cf989-75a7-4f12-be3d-643068d6d74c",
   "metadata": {},
   "outputs": [],
   "source": [
    "def search(query):\n",
    "    Entrez.email = 'storytellinginsights@gmail.com'\n",
    "    handle = Entrez.esearch(db='pubmed',\n",
    "                            sort='relevance',\n",
    "                            retmax='20',\n",
    "                            retmode='xml',\n",
    "                            term=query)\n",
    "    results = Entrez.read(handle)\n",
    "    return results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "1fa9b242-68d9-4d3c-9ef1-dc16485f18dd",
   "metadata": {},
   "outputs": [],
   "source": [
    "def fetch_details(id_list):\n",
    "    ids = ','.join(id_list)\n",
    "    Entrez.email = 'storytellinginsights@gmail.com'\n",
    "    handle = Entrez.efetch(db='pubmed',\n",
    "                           retmode='xml',\n",
    "                           id=ids)\n",
    "    results = Entrez.read(handle)\n",
    "    return results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "f1bfd630-4515-41d8-ae2a-86925531a99c",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1) {'Year': '2021', 'Month': '04', 'Day': '27'}\n",
      "1) Machine Learning and Novel Biomarkers for the Diagnosis of Alzheimer's Disease.\n",
      "1) [StringElement(\"Alzheimer's disease (AD) is a complex and severe neurodegenerative disease that still lacks effective methods of diagnosis. The current diagnostic methods of AD rely on cognitive tests, imaging techniques and cerebrospinal fluid (CSF) levels of amyloid-β1-42 (Aβ42), total tau protein and hyperphosphorylated tau (p-tau). However, the available methods are expensive and relatively invasive. Artificial intelligence techniques like machine learning tools have being increasingly used in precision diagnosis.\", attributes={'Label': 'BACKGROUND', 'NlmCategory': 'BACKGROUND'}), StringElement('We conducted a meta-analysis to investigate the machine learning and novel biomarkers for the diagnosis of AD.', attributes={'Label': 'METHODS', 'NlmCategory': 'METHODS'}), StringElement('We searched PubMed, the Cochrane Central Register of Controlled Trials, and the Cochrane Database of Systematic Reviews for reviews and trials that investigated the machine learning and novel biomarkers in diagnosis of AD.', attributes={'Label': 'METHODS', 'NlmCategory': 'METHODS'}), StringElement('In additional to Aβ and tau-related biomarkers, biomarkers according to other mechanisms of AD pathology have been investigated. Neuronal injury biomarker includes neurofiliament light (NFL). Biomarkers about synaptic dysfunction and/or loss includes neurogranin, BACE1, synaptotagmin, SNAP-25, GAP-43, synaptophysin. Biomarkers about neuroinflammation includes sTREM2, and YKL-40. Besides, d-glutamate is one of coagonists at the NMDARs. Several machine learning algorithms including support vector machine, logistic regression, random forest, and naïve Bayes) to build an optimal predictive model to distinguish patients with AD from healthy controls.', attributes={'Label': 'RESULTS', 'NlmCategory': 'RESULTS'}), StringElement('Our results revealed machine learning with novel biomarkers and multiple variables may increase the sensitivity and specificity in diagnosis of AD. Rapid and cost-effective HPLC for biomarkers and machine learning algorithms may assist physicians in diagnosing AD in outpatient clinics.', attributes={'Label': 'CONCLUSIONS', 'NlmCategory': 'CONCLUSIONS'})]\n",
      "2) {'Year': '2021', 'Month': '04', 'Day': '27'}\n",
      "2) MRI-Based Brain Tumor Classification Using Ensemble of Deep Features and Machine Learning Classifiers.\n",
      "2) ['Brain tumor classification plays an important role in clinical diagnosis and effective treatment. In this work, we propose a method for brain tumor classification using an ensemble of deep features and machine learning classifiers. In our proposed framework, we adopt the concept of transfer learning and uses several pre-trained deep convolutional neural networks to extract deep features from brain magnetic resonance (MR) images. The extracted deep features are then evaluated by several machine learning classifiers. The top three deep features which perform well on several machine learning classifiers are selected and concatenated as an ensemble of deep features which is then fed into several machine learning classifiers to predict the final output. To evaluate the different kinds of pre-trained models as a deep feature extractor, machine learning classifiers, and the effectiveness of an ensemble of deep feature for brain tumor classification, we use three different brain magnetic resonance imaging (MRI) datasets that are openly accessible from the web. Experimental results demonstrate that an ensemble of deep features can help improving performance significantly, and in most cases, support vector machine (SVM) with radial basis function (RBF) kernel outperforms other machine learning classifiers, especially for large datasets.']\n",
      "3) {'Year': '2021', 'Month': '04', 'Day': '23'}\n",
      "3) Systematic literature review of machine learning methods used in the analysis of real-world data for patient-provider decision making.\n",
      "3) [StringElement('Machine learning is a broad term encompassing a number of methods that allow the investigator to learn from the data. These methods may permit large real-world databases to be more rapidly translated to applications to inform patient-provider decision making.', attributes={'Label': 'BACKGROUND'}), StringElement('This systematic literature review was conducted to identify published observational research of employed machine learning to inform decision making at the patient-provider level. The search strategy was implemented and studies meeting eligibility criteria were evaluated by two independent reviewers. Relevant data related to study design, statistical methods and strengths and limitations were identified; study quality was assessed using a modified version of the Luo checklist.', attributes={'Label': 'METHODS'}), StringElement('A total of 34 publications from January 2014 to September 2020 were identified and evaluated for this review. There were diverse methods, statistical packages and approaches used across identified studies. The most common methods included decision tree and random forest approaches. Most studies applied internal validation but only two conducted external validation. Most studies utilized one algorithm, and only eight studies applied multiple machine learning algorithms to the data. Seven items on the Luo checklist failed to be met by more than 50% of published studies.', attributes={'Label': 'RESULTS'}), StringElement('A wide variety of approaches, algorithms, statistical software, and validation strategies were employed in the application of machine learning methods to inform patient-provider decision making. There is a need to ensure that multiple machine learning approaches are used, the model selection strategy is clearly defined, and both internal and external validation are necessary to be sure that decisions for patient care are being made with the highest quality evidence. Future work should routinely employ ensemble methods incorporating multiple machine learning algorithms.', attributes={'Label': 'CONCLUSIONS'})]\n",
      "4) {'Year': '2021', 'Month': '03', 'Day': '31'}\n",
      "4) Using Automated Machine Learning to Predict the Mortality of Patients With COVID-19: Prediction Model Development Study.\n",
      "4) [StringElement('During a pandemic, it is important for clinicians to stratify patients and decide who receives limited medical resources. Machine learning models have been proposed to accurately predict COVID-19 disease severity. Previous studies have typically tested only one machine learning algorithm and limited performance evaluation to area under the curve analysis. To obtain the best results possible, it may be important to test different machine learning algorithms to find the best prediction model.', attributes={'Label': 'BACKGROUND'}), StringElement(\"In this study, we aimed to use automated machine learning (autoML) to train various machine learning algorithms. We selected the model that best predicted patients' chances of surviving a SARS-CoV-2 infection. In addition, we identified which variables (ie, vital signs, biomarkers, comorbidities, etc) were the most influential in generating an accurate model.\", attributes={'Label': 'OBJECTIVE'}), StringElement(\"Data were retrospectively collected from all patients who tested positive for COVID-19 at our institution between March 1 and July 3, 2020. We collected 48 variables from each patient within 36 hours before or after the index time (ie, real-time polymerase chain reaction positivity). Patients were followed for 30 days or until death. Patients' data were used to build 20 machine learning models with various algorithms via autoML. The performance of machine learning models was measured by analyzing the area under the precision-recall curve (AUPCR). Subsequently, we established model interpretability via Shapley additive explanation and partial dependence plots to identify and rank variables that drove model predictions. Afterward, we conducted dimensionality reduction to extract the 10 most influential variables. AutoML models were retrained by only using these 10 variables, and the output models were evaluated against the model that used 48 variables.\", attributes={'Label': 'METHODS'}), StringElement('Data from 4313 patients were used to develop the models. The best model that was generated by using autoML and 48 variables was the stacked ensemble model (AUPRC=0.807). The two best independent models were the gradient boost machine and extreme gradient boost models, which had an AUPRC of 0.803 and 0.793, respectively. The deep learning model (AUPRC=0.73) was substantially inferior to the other models. The 10 most influential variables for generating high-performing models were systolic and diastolic blood pressure, age, pulse oximetry level, blood urea nitrogen level, lactate dehydrogenase level, D-dimer level, troponin level, respiratory rate, and Charlson comorbidity score. After the autoML models were retrained with these 10 variables, the stacked ensemble model still had the best performance (AUPRC=0.791).', attributes={'Label': 'RESULTS'}), StringElement('We used autoML to develop high-performing models that predicted the survival of patients with COVID-19. In addition, we identified important variables that correlated with mortality. This is proof of concept that autoML is an efficient, effective, and informative method for generating machine learning-based clinical decision support tools.', attributes={'Label': 'CONCLUSIONS'})]\n",
      "5) {'Year': '2021', 'Month': '02', 'Day': '04'}\n",
      "5) MegaR: an interactive R package for rapid sample classification and phenotype prediction using metagenome profiles and machine learning.\n",
      "5) [StringElement('Diverse microbiome communities drive biogeochemical processes and evolution of animals in their ecosystems. Many microbiome projects have demonstrated the power of using metagenomics to understand the structures and factors influencing the function of the microbiomes in their environments. In order to characterize the effects from microbiome composition for human health, diseases, and even ecosystems, one must first understand the relationship of microbes and their environment in different samples. Running machine learning model with metagenomic sequencing data is encouraged for this purpose, but it is not an easy task to make an appropriate machine learning model for all diverse metagenomic datasets.', attributes={'Label': 'BACKGROUND', 'NlmCategory': 'BACKGROUND'}), StringElement('We introduce MegaR, an R Shiny package and web application, to build an unbiased machine learning model effortlessly with interactive visual analysis. The MegaR employs taxonomic profiles from either whole metagenome sequencing or 16S rRNA sequencing data to develop machine learning models and classify the samples into two or more categories. It provides various options for model fine tuning throughout the analysis pipeline such as data processing, multiple machine learning techniques, model validation, and unknown sample prediction that can be used to achieve the highest prediction accuracy possible for any given dataset while still maintaining a user-friendly experience.', attributes={'Label': 'RESULTS', 'NlmCategory': 'RESULTS'}), StringElement('Metagenomic sample classification and phenotype prediction is important particularly when it applies to a diagnostic method for identifying and predicting microbe-related human diseases. MegaR provides various interactive visualizations for user to build an accurate machine-learning model without difficulty. Unknown sample prediction with a properly trained model using MegaR will enhance researchers to identify the sample property in a fast turnaround time.', attributes={'Label': 'CONCLUSIONS', 'NlmCategory': 'CONCLUSIONS'})]\n",
      "6) {'Year': '2021', 'Month': '02', 'Day': '09'}\n",
      "6) Application of Machine Learning Techniques to Predict Binding Affinity for Drug Targets: A Study of Cyclin-Dependent Kinase 2.\n",
      "6) [StringElement('The elucidation of the structure of cyclin-dependent kinase 2 (CDK2) made it possible to develop targeted scoring functions for virtual screening aimed to identify new inhibitors for this enzyme. CDK2 is a protein target for the development of drugs intended to modulate cellcycle progression and control. Such drugs have potential anticancer activities.', attributes={'Label': 'BACKGROUND', 'NlmCategory': 'BACKGROUND'}), StringElement('Our goal here is to review recent applications of machine learning methods to predict ligand- binding affinity for protein targets. To assess the predictive performance of classical scoring functions and targeted scoring functions, we focused our analysis on CDK2 structures.', attributes={'Label': 'OBJECTIVE', 'NlmCategory': 'OBJECTIVE'}), StringElement('We have experimental structural data for hundreds of binary complexes of CDK2 with different ligands, many of them with inhibition constant information. We investigate here computational methods to calculate the binding affinity of CDK2 through classical scoring functions and machine- learning models.', attributes={'Label': 'METHODS', 'NlmCategory': 'METHODS'}), StringElement('Analysis of the predictive performance of classical scoring functions available in docking programs such as Molegro Virtual Docker, AutoDock4, and Autodock Vina indicated that these methods failed to predict binding affinity with significant correlation with experimental data. Targeted scoring functions developed through supervised machine learning techniques showed a significant correlation with experimental data.', attributes={'Label': 'RESULTS', 'NlmCategory': 'RESULTS'}), StringElement('Here, we described the application of supervised machine learning techniques to generate a scoring function to predict binding affinity. Machine learning models showed superior predictive performance when compared with classical scoring functions. Analysis of the computational models obtained through machine learning could capture essential structural features responsible for binding affinity against CDK2.', attributes={'Label': 'CONCLUSION', 'NlmCategory': 'CONCLUSIONS'})]\n",
      "7) {'Year': '2021', 'Month': '02', 'Day': '05'}\n",
      "7) Machine learning methods are comparable to logistic regression techniques in predicting severe walking limitation following total knee arthroplasty.\n",
      "7) [StringElement('Machine-learning methods are flexible prediction algorithms with potential advantages over conventional regression. This study aimed to use machine learning methods to predict post-total knee arthroplasty (TKA) walking limitation, and to compare their performance with that of logistic regression.', attributes={'Label': 'PURPOSE', 'NlmCategory': 'OBJECTIVE'}), StringElement(\"From the department's clinical registry, a cohort of 4026 patients who underwent elective, primary TKA between July 2013 and July 2017 was identified. Candidate predictors included demographics and preoperative clinical, psychosocial, and outcome measures. The primary outcome was severe walking limitation at 6\\xa0months post-TKA, defined as a maximum walk time ≤\\u200915\\xa0min. Eight common regression (logistic, penalized logistic, and ordinal logistic with natural splines) and ensemble machine learning (random forest, extreme gradient boosting, and SuperLearner) methods were implemented to predict the probability of severe walking limitation. Models were compared on discrimination and calibration metrics.\", attributes={'Label': 'METHODS', 'NlmCategory': 'METHODS'}), StringElement('At 6\\xa0months post-TKA, 13% of patients had severe walking limitation. Machine learning and logistic regression models performed moderately [mean area under the ROC curves (AUC) 0.73-0.75]. Overall, the ordinal logistic regression model performed best while the SuperLearner performed best among machine learning methods, with negligible differences between them (Brier score difference, <\\u20090.001; 95% CI [-\\u20090.0025, 0.002]).', attributes={'Label': 'RESULTS', 'NlmCategory': 'RESULTS'}), StringElement('When predicting post-TKA physical function, several machine learning methods did not outperform logistic regression-in particular, ordinal logistic regression that does not assume linearity in its predictors.', attributes={'Label': 'CONCLUSIONS', 'NlmCategory': 'CONCLUSIONS'}), StringElement('Prognostic level II.', attributes={'Label': 'LEVEL OF EVIDENCE', 'NlmCategory': 'METHODS'})]\n",
      "8) {'Year': '2021', 'Month': '02', 'Day': '02'}\n",
      "8) Predicting clinically significant motor function improvement after contemporary task-oriented interventions using machine learning approaches.\n",
      "8) [StringElement('Accurate prediction of motor recovery after stroke is critical for treatment decisions and planning. Machine learning has been proposed to be a promising technique for outcome prediction because of its high accuracy and ability to process large volumes of data. It has been used to predict acute stroke recovery; however, whether machine learning would be effective for predicting rehabilitation outcomes in chronic stroke patients for common contemporary task-oriented interventions remains largely unexplored. This study aimed to determine the accuracy and performance of machine learning to predict clinically significant motor function improvements after contemporary task-oriented intervention in chronic stroke patients and identify important predictors for building machine learning prediction models.', attributes={'Label': 'BACKGROUND'}), StringElement('This study was a secondary analysis of data using two common machine learning approaches, which were the k-nearest neighbor (KNN) and artificial neural network (ANN). Chronic stroke patients (N\\u2009=\\u2009239) that received 30\\xa0h of task-oriented training including the constraint-induced movement therapy, bilateral arm training, robot-assisted therapy and mirror therapy were included. The Fugl-Meyer assessment scale (FMA) was the main outcome. Potential predictors include age, gender, side of lesion, time since stroke, baseline functional status, motor function and quality of life. We divided the data set into a training set and a test set and used the cross-validation procedure to construct machine learning models based on the training set. After the models were built, we used the test data set to evaluate the accuracy and prediction performance of the models.', attributes={'Label': 'METHODS'}), StringElement('Three important predictors were identified, which were time since stroke, baseline functional independence measure (FIM) and baseline FMA scores. Models for predicting motor function improvements were accurate. The prediction accuracy of the KNN model was 85.42% and area under the receiver operating characteristic curve (AUC-ROC) was 0.89. The prediction accuracy of the ANN model was 81.25% and the AUC-ROC was 0.77.', attributes={'Label': 'RESULTS'}), StringElement('Incorporating machine learning into clinical outcome prediction using three key predictors including time since stroke, baseline functional and motor ability may help clinicians/therapists to identify patients that are most likely to benefit from contemporary task-oriented interventions. The KNN and ANN models may be potentially useful for predicting clinically significant motor recovery in chronic stroke.', attributes={'Label': 'CONCLUSIONS'})]\n",
      "9) {'Year': '2021', 'Month': '02', 'Day': '02'}\n",
      "9) Classification of Parkinson's disease and essential tremor based on balance and gait characteristics from wearable motion sensors via machine learning techniques: a data-driven approach.\n",
      "9) [StringElement(\"Parkinson's disease (PD) and essential tremor (ET) are movement disorders that can have similar clinical characteristics including tremor and gait difficulty. These disorders can be misdiagnosed leading to delay in appropriate treatment. The aim of the study was to determine whether balance and gait variables obtained with wearable inertial motion sensors can be utilized to differentiate between PD and ET using machine learning. Additionally, we compared classification performances of several machine learning models.\", attributes={'Label': 'BACKGROUND'}), StringElement('This retrospective study included balance and gait variables collected during the instrumented stand and walk test from people with PD (n\\u2009=\\u2009524) and with ET (n\\u2009=\\u200943). Performance of several machine learning techniques including neural networks, support vector machine, k-nearest neighbor, decision tree, random forest, and gradient boosting, were compared with a dummy model or logistic regression using F1-scores.', attributes={'Label': 'METHODS'}), StringElement('Machine learning models classified PD and ET based on balance and gait characteristics better than the dummy model (F1-score\\u2009=\\u20090.48) or logistic regression (F1-score\\u2009=\\u20090.53). The highest F1-score was 0.61 of neural network, followed by 0.59 of gradient boosting, 0.56 of random forest, 0.55 of support vector machine, 0.53 of decision tree, and 0.49 of k-nearest neighbor.', attributes={'Label': 'RESULTS'}), StringElement('This study demonstrated the utility of machine learning models to classify different movement disorders based on balance and gait characteristics collected from wearable sensors. Future studies using a well-balanced data set are needed to confirm the potential clinical utility of machine learning models to discern between PD and ET.', attributes={'Label': 'CONCLUSIONS'})]\n",
      "10) {'Year': '2020', 'Month': '09', 'Day': '29'}\n",
      "10) Evolution and impact of bias in human and machine learning algorithm interaction.\n",
      "10) [\"Traditionally, machine learning algorithms relied on reliable labels from experts to build predictions. More recently however, algorithms have been receiving data from the general population in the form of labeling, annotations, etc. The result is that algorithms are subject to bias that is born from ingesting unchecked information, such as biased samples and biased labels. Furthermore, people and algorithms are increasingly engaged in interactive processes wherein neither the human nor the algorithms receive unbiased data. Algorithms can also make biased predictions, leading to what is now known as algorithmic bias. On the other hand, human's reaction to the output of machine learning methods with algorithmic bias worsen the situations by making decision based on biased information, which will probably be consumed by algorithms later. Some recent research has focused on the ethical and moral implication of machine learning algorithmic bias on society. However, most research has so far treated algorithmic bias as a static factor, which fails to capture the dynamic and iterative properties of bias. We argue that algorithmic bias interacts with humans in an iterative manner, which has a long-term effect on algorithms' performance. For this purpose, we present an iterated-learning framework that is inspired from human language evolution to study the interaction between machine learning algorithms and humans. Our goal is to study two sources of bias that interact: the process by which people select information to label (human action); and the process by which an algorithm selects the subset of information to present to people (iterated algorithmic bias mode). We investigate three forms of iterated algorithmic bias (personalization filter, active learning, and random) and how they affect the performance of machine learning algorithms by formulating research questions about the impact of each type of bias. Based on statistical analyses of the results of several controlled experiments, we found that the three different iterated bias modes, as well as initial training data class imbalance and human action, do affect the models learned by machine learning algorithms. We also found that iterated filter bias, which is prominent in personalized user interfaces, can lead to more inequality in estimated relevance and to a limited human ability to discover relevant data. Our findings indicate that the relevance blind spot (items from the testing set whose predicted relevance probability is less than 0.5 and who thus risk being hidden from humans) amounted to 4% of all relevant items when using a content-based filter that predicts relevant items. A similar simulation using a real-life rating data set found that the same filter resulted in a blind spot size of 75% of the relevant testing set.\"]\n",
      "11) {'Year': '2021', 'Month': '04', 'Day': '20'}\n",
      "11) Incorporating Machine Learning into Established Bioinformatics Frameworks.\n",
      "11) ['The exponential growth of biomedical data in recent years has urged the application of numerous machine learning techniques to address emerging problems in biology and clinical research. By enabling the automatic feature extraction, selection, and generation of predictive models, these methods can be used to efficiently study complex biological systems. Machine learning techniques are frequently integrated with bioinformatic methods, as well as curated databases and biological networks, to enhance training and validation, identify the best interpretable features, and enable feature and model investigation. Here, we review recently developed methods that incorporate machine learning within the same framework with techniques from molecular evolution, protein structure analysis, systems biology, and disease genomics. We outline the challenges posed for machine learning, and, in particular, deep learning in biomedicine, and suggest unique opportunities for machine learning techniques integrated with established bioinformatics approaches to overcome some of these challenges.']\n",
      "12) {'Year': '2021', 'Month': '03', 'Day': '18'}\n",
      "12) Machine learning prediction of stone-free success in patients with urinary stone after treatment of shock wave lithotripsy.\n",
      "12) [StringElement('The aims of this study were to determine the predictive value of decision support analysis for the shock wave lithotripsy (SWL) success rate and to analyze the data obtained from patients who underwent SWL to assess the factors influencing the outcome by using machine learning methods.', attributes={'Label': 'BACKGROUND', 'NlmCategory': 'BACKGROUND'}), StringElement('We retrospectively reviewed the medical records of 358 patients who underwent SWL for urinary stone (kidney and upper-ureter stone) between 2015 and 2018 and evaluated the possible prognostic features, including patient population characteristics, urinary stone characteristics on a non-contrast, computed tomographic image. We performed 80% training set and 20% test set for the predictions of success and mainly used decision tree-based machine learning algorithms, such as random forest (RF), extreme gradient boosting trees (XGBoost), and light gradient boosting method (LightGBM).', attributes={'Label': 'METHODS', 'NlmCategory': 'METHODS'}), StringElement('In machine learning analysis, the prediction accuracies for stone-free were 86.0, 87.5, and 87.9%, and those for one-session success were 78.0, 77.4, and 77.0% using RF, XGBoost, and LightGBM, respectively. In predictions for stone-free, LightGBM yielded the best accuracy and RF yielded the best one in those for one-session success among those methods. The sensitivity and specificity values for machine learning analytics are (0.74 to 0.78 and 0.92 to 0.93) for stone-free and (0.79 to 0.81 and 0.74 to 0.75) for one-session success, respectively. The area under curve (AUC) values for machine learning analytics are (0.84 to 0.85) for stone-free and (0.77 to 0.78) for one-session success and their 95% confidence intervals (CIs) are (0.730 to 0.933) and (0.673 to 0.866) in average of methods, respectively.', attributes={'Label': 'RESULTS', 'NlmCategory': 'RESULTS'}), StringElement('We applied a selected machine learning analysis to predict the result after treatment of SWL for urinary stone. About 88% accurate machine learning based predictive model was evaluated. The importance of machine learning algorithm can give matched insights to domain knowledge on effective and influential factors for SWL success outcomes.', attributes={'Label': 'CONCLUSIONS', 'NlmCategory': 'CONCLUSIONS'})]\n",
      "13) {'Year': '2021', 'Month': '04', 'Day': '23'}\n",
      "13) Predicting Australian Adults at High Risk of Cardiovascular Disease Mortality Using Standard Risk Factors and Machine Learning.\n",
      "13) ['Effective cardiovascular disease (CVD) prevention relies on timely identification and intervention for individuals at risk. Conventional formula-based techniques have been demonstrated to over- or under-predict the risk of CVD in the Australian population. This study assessed the ability of machine learning models to predict CVD mortality risk in the Australian population and compare performance with the well-established Framingham model. Data is drawn from three Australian cohort studies: the North West Adelaide Health Study (NWAHS), the Australian Diabetes, Obesity, and Lifestyle study, and the Melbourne Collaborative Cohort Study (MCCS). Four machine learning models for predicting 15-year CVD mortality risk were developed and compared to the 2008 Framingham model. Machine learning models performed significantly better compared to the Framingham model when applied to the three Australian cohorts. Machine learning based models improved prediction by 2.7% to 5.2% across three Australian cohorts. In an aggregated cohort, machine learning models improved prediction by up to 5.1% (area-under-curve (AUC) 0.852, 95% CI 0.837-0.867). Net reclassification improvement (NRI) was up to 26% with machine learning models. Machine learning based models also showed improved performance when stratified by sex and diabetes status. Results suggest a potential for improving CVD risk prediction in the Australian population using machine learning models.']\n",
      "14) {'Year': '2021', 'Month': '04', 'Day': '26'}\n",
      "14) Machine learning and artificial intelligence in haematology.\n",
      "14) ['Digitalization of the medical record and integration of genomic methods into clinical practice have resulted in an unprecedented wealth of data. Machine learning is a subdomain of artificial intelligence that attempts to computationally extract meaningful insights from complex data structures. Applications of machine learning in haematological scenarios are steadily increasing. However, basic concepts are often unfamiliar to clinicians and investigators. The purpose of this review is to provide readers with tools to interpret and critically appraise machine learning literature. We begin with the elucidation of standard terminology and then review examples in haematology. Guidelines for designing and evaluating machine-learning studies are provided. Finally, we discuss limitations of the machine-learning approach.']\n",
      "15) {'Year': '2021', 'Month': '03', 'Day': '01'}\n",
      "15) Machine learning techniques for biomedical image segmentation: An overview of technical aspects and introduction to state-of-art applications.\n",
      "15) ['In recent years, significant progress has been made in developing more accurate and efficient machine learning algorithms for segmentation of medical and natural images. In this review article, we highlight the imperative role of machine learning algorithms in enabling efficient and accurate segmentation in the field of medical imaging. We specifically focus on several key studies pertaining to the application of machine learning methods to biomedical image segmentation. We review classical machine learning algorithms such as Markov random fields, k-means clustering, random forest, etc. Although such classical learning models are often less accurate compared to the deep-learning techniques, they are often more sample efficient and have a less complex structure. We also review different deep-learning architectures, such as the artificial neural networks (ANNs), the convolutional neural networks (CNNs), and the recurrent neural networks (RNNs), and present the segmentation results attained by those learning models that were published in the past 3\\xa0yr. We highlight the successes and limitations of each machine learning paradigm. In addition, we discuss several challenges related to the training of different machine learning models, and we present some heuristics to address those challenges.']\n",
      "16) {'Year': '2021', 'Month': '02', 'Day': '19'}\n",
      "16) How Good Is Machine Learning in Predicting All-Cause 30-Day Hospital Readmission? Evidence From Administrative Data.\n",
      "16) [StringElement('Hospital readmission is a main cost driver for healthcare systems, but existing works often had poor or moderate predictive results. Although the available information differs in different studies, improving prediction is different from the search for important explanatory variables. With large sample size and abundant information, this study explores state-of-the-art machine-learning algorithms and shows their performance in prediction.', attributes={'Label': 'OBJECTIVES'}), StringElement('Using administrative data on 1\\u2009631\\u2009611 hospital stays from Quebec between 1995 and 2012, we predict the probability of 30-day readmission at hospital admission and discharge. We compare the performance between traditional logistic regression, logistic regression with penalization, and more recent machine-learning algorithms such as random forest, deep learning, and extreme gradient boosting.', attributes={'Label': 'METHODS'}), StringElement('After a 10-fold cross-validation on the training set (80% of the data), machine learning produced very good results on a separate hold-out test set (20% of the data). The importance of explanatory variables is not the same for different algorithms. The area under receiver operating characteristic curve (AUC) reached above 0.79 at hospital admission and above 0.88 at hospital discharge. Diagnostic codes, which include many different categories, are among the most predictive variables. Logistic regression with penalization also produced good results, but a standard logistic regression failed without penalization. The good results are confirmed by calibration curves.', attributes={'Label': 'RESULTS'}), StringElement('Although the identification of those at highest risk of readmission is just 1 step to preventing hospital readmissions, 30-day readmission is highly predictable with machine learning.', attributes={'Label': 'CONCLUSION'})]\n",
      "17) {'Year': '2021', 'Month': '03', 'Day': '05'}\n",
      "17) Comparison of nomogram with machine learning techniques for prediction of overall survival in patients with tongue cancer.\n",
      "17) [StringElement('The prediction of overall survival in tongue cancer is important for planning of personalized care and patient counselling.', attributes={'Label': 'BACKGROUND'}), StringElement('This study compares the performance of a nomogram with a machine learning model to predict overall survival in tongue cancer. The nomogram and machine learning model were built using a large data set from the Surveillance, Epidemiology, and End Results (SEER) program database. The comparison is necessary to provide the clinicians with a comprehensive, practical, and most accurate assistive system to predict overall survival of this patient population.', attributes={'Label': 'OBJECTIVES'}), StringElement('The data set used included the records of 7596 tongue cancer patients. The considered machine learning algorithms were logistic regression, support vector machine, Bayes point machine, boosted decision tree, decision forest, and decision jungle. These algorithms were mainly evaluated in terms of the areas under the receiver-operating characteristic (ROC) curve (AUC) and accuracy values. The performance of the algorithm that produced the best result was compared with a nomogram to predict overall survival in tongue cancer patients.', attributes={'Label': 'METHODS'}), StringElement(\"The boosted decision-tree algorithm outperformed other algorithms. When compared with a nomogram using external validation data, the boosted decision tree produced an accuracy of 88.7% while the nomogram showed an accuracy of 60.4%. In addition, it was found that age of patient, T stage, radiotherapy, and the surgical resection were the most prominent features with significant influence on the machine learning model's performance to predict overall survival.\", attributes={'Label': 'RESULTS'}), StringElement(\"The machine learning model provides more personalized and reliable prognostic information of tongue cancer than the nomogram. However, the level of transparency offered by the nomogram in estimating patients' outcomes seems more confident and strengthened the principle of shared decision making between the patient and clinician. Therefore, a combination of a nomogram - machine learning (NomoML) predictive model may help to improve care, provides information to patients, and facilitates the clinicians in making tongue cancer management-related decisions.\", attributes={'Label': 'CONCLUSION'})]\n",
      "18) {'Year': '2021', 'Month': '04', 'Day': '12'}\n",
      "18) The Application of a Machine Learning-Based Brain Magnetic Resonance Imaging Approach in Major Depression.\n",
      "18) ['Major depressive disorder (MDD) shows a high prevalence and is associated with increased disability. While traditional studies aimed to investigate global characteristic neurobiological substrates of MDD, machine learning-based approaches focus on individual people rather than a group. Therefore, machine learning has been increasingly conducted and applied to clinical practice. Several previous neuroimaging studies used machine learning for stratifying MDD patients from healthy controls as well as in differentially diagnosing MDD apart from other psychiatric disorders. Also, machine learning has been used to predict treatment response using magnetic resonance imaging (MRI) results. Despite the recent accomplishments of machine learning-based MRI studies, small sample sizes and the heterogeneity of the depression group limit the generalizability of a machine learning-based predictive model. Future neuroimaging studies should integrate various materials such as genetic, peripheral, and clinical phenotypes for more accurate predictability of diagnosis and treatment response.']\n",
      "19) {'Year': '2021', 'Month': '04', 'Day': '27'}\n",
      "19) Prediction of the Age at Onset of Spinocerebellar Ataxia Type 3 with Machine Learning.\n",
      "19) [StringElement(\"In polyglutamine (polyQ) disease, the investigation of the prediction of a patient's age at onset (AAO) facilitates the development of disease-modifying intervention and underpins the delay of disease onset and progression. Few polyQ disease studies have evaluated AAO predicted by machine-learning algorithms and linear regression methods.\", attributes={'Label': 'BACKGROUND'}), StringElement('The objective of this study was to develop a machine-learning model for AAO prediction in the largest spinocerebellar ataxia type 3/Machado-Joseph disease (SCA3/MJD) population from mainland China.', attributes={'Label': 'OBJECTIVE'}), StringElement('In this observational study, we introduced an innovative approach by systematically comparing the performance of 7 machine-learning algorithms with linear regression to explore AAO prediction in SCA3/MJD using CAG expansions of 10 polyQ-related genes, sex, and parental origin.', attributes={'Label': 'METHODS'}), StringElement('Similar prediction performance of testing set and training set in each models were identified and few overfitting of training data was observed. Overall, the machine-learning-based XGBoost model exhibited the most favorable performance in AAO prediction over the traditional linear regression method and other 6 machine-learning algorithms for the training set and testing set. The optimal XGBoost model achieved mean absolute error, root mean square error, and median absolute error of 5.56, 7.13, 4.15\\u2009years, respectively, in testing set 1, with mean absolute error (4.78\\u2009years), root mean square error (6.31\\u2009years), and median absolute error (3.59\\u2009years) in testing set 2.', attributes={'Label': 'RESULTS'}), StringElement('Machine-learning algorithms can be used to predict AAO in patients with SCA3/MJD. The optimal XGBoost algorithm can provide a good reference for the establishment and optimization of prediction models for SCA3/MJD or other polyQ diseases. © 2020 International Parkinson and Movement Disorder Society.', attributes={'Label': 'CONCLUSION'})]\n",
      "20) {'Year': '2021', 'Month': '04', 'Day': '12'}\n",
      "20) R.ROSETTA: an interpretable machine learning framework.\n",
      "20) [StringElement('Machine learning involves strategies and algorithms that may assist bioinformatics analyses in terms of data mining and knowledge discovery. In several applications, viz. in Life Sciences, it is often more important to understand how a prediction was obtained rather than knowing what prediction was made. To this end so-called interpretable machine learning has been recently advocated. In this study, we implemented an interpretable machine learning package based on the rough set theory. An important aim of our work was provision of statistical properties of the models and their components.', attributes={'Label': 'BACKGROUND', 'NlmCategory': 'BACKGROUND'}), StringElement('We present the R.ROSETTA package, which is an R wrapper of ROSETTA framework. The original ROSETTA functions have been improved and adapted to the R programming environment. The package allows for building and analyzing non-linear interpretable machine learning models. R.ROSETTA gathers combinatorial statistics via rule-based modelling for accessible and transparent results, well-suited for adoption within the greater scientific community. The package also provides statistics and visualization tools that facilitate minimization of analysis bias and noise. The R.ROSETTA package is freely available at https://github.com/komorowskilab/R.ROSETTA . To illustrate the usage of the package, we applied it to a transcriptome dataset from an autism case-control study. Our tool provided hypotheses for potential co-predictive mechanisms among features that discerned phenotype classes. These co-predictors represented neurodevelopmental and autism-related genes.', attributes={'Label': 'RESULTS', 'NlmCategory': 'RESULTS'}), StringElement('R.ROSETTA provides new insights for interpretable machine learning analyses and knowledge-based systems. We demonstrated that our package facilitated detection of dependencies for autism-related genes. Although the sample application of R.ROSETTA illustrates transcriptome data analysis, the package can be used to analyze any data organized in decision tables.', attributes={'Label': 'CONCLUSIONS', 'NlmCategory': 'CONCLUSIONS'})]\n"
     ]
    }
   ],
   "source": [
    "if __name__ == '__main__':\n",
    "    results = search('machine learning' -datetype PDAT -mindate 2020 -maxdate 2021 -sort 'Relevance')\n",
    "    id_list = results['IdList']\n",
    "    papers = fetch_details(id_list)\n",
    "    for i, paper in enumerate(papers['PubmedArticle']):\n",
    "        print(\"{}) {}\".format(i+1, paper['MedlineCitation']['DateCompleted']))\n",
    "        print(\"{}) {}\".format(i+1, paper['MedlineCitation']['Article']['ArticleTitle']))\n",
    "        print(\"{}) {}\".format(i+1, paper['MedlineCitation']['Article']['Abstract']['AbstractText']))\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "3fbd2f33-cf31-4df0-9f15-eaca11cd4e93",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "  \"MedlineCitation\": {\n",
      "    \"OtherID\": [],\n",
      "    \"GeneralNote\": [],\n",
      "    \"OtherAbstract\": [],\n",
      "    \"SpaceFlightMission\": [],\n",
      "    \"CitationSubset\": [\n",
      "      \"IM\"\n",
      "    ],\n",
      "    \"KeywordList\": [\n",
      "      [\n",
      "        \"AI\",\n",
      "        \"Alzheimer\\u2019s disease\",\n",
      "        \"biomarker\",\n",
      "        \"deep learning\",\n",
      "        \"machine learning\"\n",
      "      ]\n",
      "    ],\n",
      "    \"PMID\": \"33803217\",\n",
      "    \"DateCompleted\": {\n",
      "      \"Year\": \"2021\",\n",
      "      \"Month\": \"04\",\n",
      "      \"Day\": \"27\"\n",
      "    },\n",
      "    \"DateRevised\": {\n",
      "      \"Year\": \"2021\",\n",
      "      \"Month\": \"04\",\n",
      "      \"Day\": \"27\"\n",
      "    },\n",
      "    \"Article\": {\n",
      "      \"ELocationID\": [\n",
      "        \"2761\",\n",
      "        \"10.3390/ijms22052761\"\n",
      "      ],\n",
      "      \"Language\": [\n",
      "        \"eng\"\n",
      "      ],\n",
      "      \"ArticleDate\": [\n",
      "        {\n",
      "          \"Year\": \"2021\",\n",
      "          \"Month\": \"03\",\n",
      "          \"Day\": \"09\"\n",
      "        }\n",
      "      ],\n",
      "      \"Journal\": {\n",
      "        \"ISSN\": \"1422-0067\",\n",
      "        \"JournalIssue\": {\n",
      "          \"Volume\": \"22\",\n",
      "          \"Issue\": \"5\",\n",
      "          \"PubDate\": {\n",
      "            \"Year\": \"2021\",\n",
      "            \"Month\": \"Mar\",\n",
      "            \"Day\": \"09\"\n",
      "          }\n",
      "        },\n",
      "        \"Title\": \"International journal of molecular sciences\",\n",
      "        \"ISOAbbreviation\": \"Int J Mol Sci\"\n",
      "      },\n",
      "      \"ArticleTitle\": \"Machine Learning and Novel Biomarkers for the Diagnosis of Alzheimer's Disease.\",\n",
      "      \"Abstract\": {\n",
      "        \"AbstractText\": [\n",
      "          \"Alzheimer's disease (AD) is a complex and severe neurodegenerative disease that still lacks effective methods of diagnosis. The current diagnostic methods of AD rely on cognitive tests, imaging techniques and cerebrospinal fluid (CSF) levels of amyloid-\\u03b21-42 (A\\u03b242), total tau protein and hyperphosphorylated tau (p-tau). However, the available methods are expensive and relatively invasive. Artificial intelligence techniques like machine learning tools have being increasingly used in precision diagnosis.\",\n",
      "          \"We conducted a meta-analysis to investigate the machine learning and novel biomarkers for the diagnosis of AD.\",\n",
      "          \"We searched PubMed, the Cochrane Central Register of Controlled Trials, and the Cochrane Database of Systematic Reviews for reviews and trials that investigated the machine learning and novel biomarkers in diagnosis of AD.\",\n",
      "          \"In additional to A\\u03b2 and tau-related biomarkers, biomarkers according to other mechanisms of AD pathology have been investigated. Neuronal injury biomarker includes neurofiliament light (NFL). Biomarkers about synaptic dysfunction and/or loss includes neurogranin, BACE1, synaptotagmin, SNAP-25, GAP-43, synaptophysin. Biomarkers about neuroinflammation includes sTREM2, and YKL-40. Besides, d-glutamate is one of coagonists at the NMDARs. Several machine learning algorithms including support vector machine, logistic regression, random forest, and na\\u00efve Bayes) to build an optimal predictive model to distinguish patients with AD from healthy controls.\",\n",
      "          \"Our results revealed machine learning with novel biomarkers and multiple variables may increase the sensitivity and specificity in diagnosis of AD. Rapid and cost-effective HPLC for biomarkers and machine learning algorithms may assist physicians in diagnosing AD in outpatient clinics.\"\n",
      "        ]\n",
      "      },\n",
      "      \"AuthorList\": [\n",
      "        {\n",
      "          \"Identifier\": [\n",
      "            \"0000-0003-2949-5855\"\n",
      "          ],\n",
      "          \"AffiliationInfo\": [\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Institute of Clinical Medical Science, China Medical University, Taichung 40402, Taiwan.\"\n",
      "            },\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Department of Psychiatry & Brain Disease Research Center, China Medical University Hospital, Taichung 40402, Taiwan.\"\n",
      "            },\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"An Nan Hospital, China Medical University, Tainan 709025, Taiwan.\"\n",
      "            }\n",
      "          ],\n",
      "          \"LastName\": \"Chang\",\n",
      "          \"ForeName\": \"Chun-Hung\",\n",
      "          \"Initials\": \"CH\"\n",
      "        },\n",
      "        {\n",
      "          \"Identifier\": [],\n",
      "          \"AffiliationInfo\": [\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Institute of Clinical Medical Science, China Medical University, Taichung 40402, Taiwan.\"\n",
      "            },\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Graduate Institute of Biomedical Sciences, China Medical University, Taichung 40402, Taiwan.\"\n",
      "            },\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Kaohsiung Chang Gung Memorial Hospital, Chang Gung University College of Medicine, Kaohsiung 83301, Taiwan.\"\n",
      "            },\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"School of Medicine, Chang Gung University, Taoyuan 83301, Taiwan.\"\n",
      "            }\n",
      "          ],\n",
      "          \"LastName\": \"Lin\",\n",
      "          \"ForeName\": \"Chieh-Hsin\",\n",
      "          \"Initials\": \"CH\"\n",
      "        },\n",
      "        {\n",
      "          \"Identifier\": [\n",
      "            \"0000-0003-2162-8174\"\n",
      "          ],\n",
      "          \"AffiliationInfo\": [\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Institute of Clinical Medical Science, China Medical University, Taichung 40402, Taiwan.\"\n",
      "            },\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Department of Psychiatry & Brain Disease Research Center, China Medical University Hospital, Taichung 40402, Taiwan.\"\n",
      "            },\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Graduate Institute of Biomedical Sciences, China Medical University, Taichung 40402, Taiwan.\"\n",
      "            },\n",
      "            {\n",
      "              \"Identifier\": [],\n",
      "              \"Affiliation\": \"Department of Psychology, College of Medical and Health Sciences, Asia University, Taichung 41354, Taiwan.\"\n",
      "            }\n",
      "          ],\n",
      "          \"LastName\": \"Lane\",\n",
      "          \"ForeName\": \"Hsien-Yuan\",\n",
      "          \"Initials\": \"HY\"\n",
      "        }\n",
      "      ],\n",
      "      \"PublicationTypeList\": [\n",
      "        \"Journal Article\",\n",
      "        \"Systematic Review\"\n",
      "      ]\n",
      "    },\n",
      "    \"MedlineJournalInfo\": {\n",
      "      \"Country\": \"Switzerland\",\n",
      "      \"MedlineTA\": \"Int J Mol Sci\",\n",
      "      \"NlmUniqueID\": \"101092791\",\n",
      "      \"ISSNLinking\": \"1422-0067\"\n",
      "    },\n",
      "    \"ChemicalList\": [\n",
      "      {\n",
      "        \"RegistryNumber\": \"0\",\n",
      "        \"NameOfSubstance\": \"Biomarkers\"\n",
      "      }\n",
      "    ],\n",
      "    \"MeshHeadingList\": [\n",
      "      {\n",
      "        \"QualifierName\": [],\n",
      "        \"DescriptorName\": \"Aged\"\n",
      "      },\n",
      "      {\n",
      "        \"QualifierName\": [\n",
      "          \"diagnosis\",\n",
      "          \"metabolism\"\n",
      "        ],\n",
      "        \"DescriptorName\": \"Alzheimer Disease\"\n",
      "      },\n",
      "      {\n",
      "        \"QualifierName\": [\n",
      "          \"metabolism\"\n",
      "        ],\n",
      "        \"DescriptorName\": \"Biomarkers\"\n",
      "      },\n",
      "      {\n",
      "        \"QualifierName\": [],\n",
      "        \"DescriptorName\": \"Chromatography, High Pressure Liquid\"\n",
      "      },\n",
      "      {\n",
      "        \"QualifierName\": [],\n",
      "        \"DescriptorName\": \"Diagnosis, Computer-Assisted\"\n",
      "      },\n",
      "      {\n",
      "        \"QualifierName\": [],\n",
      "        \"DescriptorName\": \"Female\"\n",
      "      },\n",
      "      {\n",
      "        \"QualifierName\": [],\n",
      "        \"DescriptorName\": \"Humans\"\n",
      "      },\n",
      "      {\n",
      "        \"QualifierName\": [],\n",
      "        \"DescriptorName\": \"Machine Learning\"\n",
      "      },\n",
      "      {\n",
      "        \"QualifierName\": [],\n",
      "        \"DescriptorName\": \"Middle Aged\"\n",
      "      }\n",
      "    ]\n",
      "  },\n",
      "  \"PubmedData\": {\n",
      "    \"ReferenceList\": [\n",
      "      {\n",
      "        \"Reference\": [\n",
      "          {\n",
      "            \"Citation\": \"Front Neurosci. 2013 Aug 29;7:155\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"24009551\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Alzheimers Dement. 2019 Jun;15(6):729-741\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"31047857\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Front Psychiatry. 2018 Nov 06;9:566\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"30459659\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Brain Imaging Behav. 2020 Apr;14(2):451-459\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"31102169\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Pharmacol Biochem Behav. 2012 Feb;100(4):656-64\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"21889952\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Front Aging Neurosci. 2019 Aug 20;11:220\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"31481890\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Ann Neurol. 2016 Aug;80(2):247-58\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"27323247\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Behav Brain Res. 2003 Mar 18;140(1-2):1-47\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"12644276\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Alzheimers Dement. 2008 Jan;4(1):38-48\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"18631949\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Pers Med. 2020 Dec 26;11(1):\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"33375220\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"N Engl J Med. 2009 Jul 16;361(3):302-3\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"19605837\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"PLoS Med. 2018 Jan 25;15(1):e1002482\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"29370177\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Alzheimers Dis. 2017;57(4):1041-1048\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"27662322\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Methods Mol Biol. 2018;1750:363-370\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"29512086\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Neurobiol Aging. 2009 Jul;30(7):1026-36\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"19376612\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Neuroimage Clin. 2018 Mar 10;18:802-813\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"29876266\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Epidemiology. 2015 Jan;26(1):e4-6\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"25437325\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Hum Brain Mapp. 2020 Oct 1;41(14):4127-4147\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"32614505\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Neuroimaging. 2015 Sep-Oct;25(5):738-47\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"25644739\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Neurosci. 2011 May 4;31(18):6627-38\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"21543591\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Psychopharmacol. 2021 Mar;35(3):265-272\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"33586518\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Nat Rev Neurol. 2010 Mar;6(3):131-44\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"20157306\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Alzheimers Dis. 2020;73(4):1385-1405\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"31958093\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Alzheimers Dement. 2019 Jun;15(6):817-827\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"31078433\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Neuropathol Exp Neurol. 2012 Apr;71(4):266-73\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"22437338\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Nat Rev Neurol. 2010 Feb;6(2):67-77\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"20139996\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Alzheimers Dement. 2011 May;7(3):257-62\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"21514247\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Front Bioeng Biotechnol. 2020 Jun 04;8:569\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"32582679\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Nat Commun. 2017 Oct 31;8(1):1214\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"29089479\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Phys Chem Chem Phys. 2021 Jan 21;23(2):1197-1214\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"33355332\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Cochrane Database Syst Rev. 2017 Mar 22;3:CD010803\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"28328043\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Int J Geriatr Psychiatry. 2003 Sep;18(Suppl 1):S33-40\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"12973748\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Neurol. 2020 Aug;267(8):2475-2477\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"32638106\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Alzheimers Dement. 2017 Apr;13(4):e1-e85\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"28342697\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Biomedicines. 2020 Oct 15;8(10):\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"33076333\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Brain. 2016 May;139(Pt 5):1539-50\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"26936940\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Lancet Neurol. 2017 Jan;16(1):4-5\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"27979354\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Amino Acids. 2017 Sep;49(9):1521-1533\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"28681245\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Ann Neurol. 2014 Apr;75(4):597-601\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"24585367\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Curr Opin Clin Nutr Metab Care. 2013 Jan;16(1):72-5\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"23041616\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Acta Neuropathol. 2006 Oct;112(4):389-404\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"16906426\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Pharmacol Biochem Behav. 2019 Oct;185:172760\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"31422081\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Pers Med. 2020 Sep 04;10(3):\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"32899797\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Brain. 2020 Jun 1;143(6):1920-1933\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"32357201\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Neurosci. 2007 Mar 14;27(11):2866-75\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"17360908\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Amino Acids. 2012 Nov;43(5):1809-10\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"22986446\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"J Pers Med. 2020 Sep 08;10(3):\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"32911755\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Amino Acids. 1998;15(3):263-9\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"9871505\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"IEEE Trans Biomed Eng. 2019 Oct;66(10):2924-2935\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"30762522\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Annu Rev Clin Psychol. 2018 May 7;14:91-118\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"29401044\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"FEBS J. 2008 Jul;275(14):3514-26\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"18564180\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Med Clin North Am. 2019 Mar;103(2):263-293\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"30704681\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Transl Psychiatry. 2015 May 05;5:e561\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"25942042\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Alzheimers Dement (N Y). 2019 Dec 18;5:933-938\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"31890857\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Sci Rep. 2017 Nov 1;7(1):14849\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"29093468\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Behav Brain Res. 2018 May 15;344:103-109\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"29454006\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Neuron. 2009 Jun 25;62(6):788-801\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"19555648\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Int J Mol Sci. 2020 Feb 01;21(3):\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"32024055\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Alzheimers Dement. 2011 May;7(3):263-9\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"21514250\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Neural Regen Res. 2021 Jan;16(1):73-79\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"32788450\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"BMC Bioinformatics. 2020 Dec 28;21(Suppl 21):496\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"33371874\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Nature. 2004 Aug 5;430(7000):631-9\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"15295589\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Acta Neuropathol. 2018 Nov;136(5):663-689\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"30349969\"\n",
      "            ]\n",
      "          },\n",
      "          {\n",
      "            \"Citation\": \"Mol Imaging. 2019 Jan-Dec;18:1536012119869070\",\n",
      "            \"ArticleIdList\": [\n",
      "              \"31429375\"\n",
      "            ]\n",
      "          }\n",
      "        ],\n",
      "        \"ReferenceList\": []\n",
      "      }\n",
      "    ],\n",
      "    \"History\": [\n",
      "      {\n",
      "        \"Year\": \"2021\",\n",
      "        \"Month\": \"01\",\n",
      "        \"Day\": \"31\"\n",
      "      },\n",
      "      {\n",
      "        \"Year\": \"2021\",\n",
      "        \"Month\": \"03\",\n",
      "        \"Day\": \"02\"\n",
      "      },\n",
      "      {\n",
      "        \"Year\": \"2021\",\n",
      "        \"Month\": \"03\",\n",
      "        \"Day\": \"05\"\n",
      "      },\n",
      "      {\n",
      "        \"Year\": \"2021\",\n",
      "        \"Month\": \"4\",\n",
      "        \"Day\": \"3\",\n",
      "        \"Hour\": \"1\",\n",
      "        \"Minute\": \"13\"\n",
      "      },\n",
      "      {\n",
      "        \"Year\": \"2021\",\n",
      "        \"Month\": \"4\",\n",
      "        \"Day\": \"4\",\n",
      "        \"Hour\": \"6\",\n",
      "        \"Minute\": \"0\"\n",
      "      },\n",
      "      {\n",
      "        \"Year\": \"2021\",\n",
      "        \"Month\": \"4\",\n",
      "        \"Day\": \"28\",\n",
      "        \"Hour\": \"6\",\n",
      "        \"Minute\": \"0\"\n",
      "      }\n",
      "    ],\n",
      "    \"PublicationStatus\": \"epublish\",\n",
      "    \"ArticleIdList\": [\n",
      "      \"33803217\",\n",
      "      \"ijms22052761\",\n",
      "      \"10.3390/ijms22052761\",\n",
      "      \"PMC7963160\"\n",
      "    ]\n",
      "  }\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "print(json.dumps(papers['PubmedArticle'][0], indent=2))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ba4338d2-b5cb-4c4a-8a72-0c025da8f521",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
